posthog/models/cohort/cohort.py

```diff
@@ -215,7 +226,9 @@ def calculate_people_ch(self, pending_version: int, *, initiating_user_id: Optio
 
             self.last_calculation = timezone.now()
             self.errors_calculating = 0
-        except Exception:
+
+            COHORT_SUCCESSFUL_CALCULATIONS_COUNTER.inc()
```

Comment:
ooh thanks for pointing this out, you're right, I can use those to graph success/errors [here](https://grafana.prod-us.posthog.dev/d/bduzbpstwo16oe/cohorts?folderUid=cd14e318-ce4f-4feb-a21e-835f015ff8c6&tab=queries&from=now-24h&to=now&timezone=browser&showCategory=Legend)

We only call them from celery. Will update PR to only log backlog and send errors to sentry, so that we can dig into failures